deploy/terraform/terraform-units/modules/sap_deployer/variables_local.tf

```
@@ -13,42 +51,49 @@ locals {
   // Default option(s):
   enable_secure_transfer = try(var.options.enable_secure_transfer, true)
 
+  // Resource group and location
+
+  region             = try(var.infrastructure.region, "")
+  landscape          = try(var.infrastructure.landscape, "global")
```

this need be a required input, just like region.

```suggestion
  landscape          = try(var.infrastructure.landscape, "")
```